INSTALL.md

```diff
@@ -41,6 +41,8 @@ Ginkgo adds the following additional switches to control what is being built:
     `OFF` otherwise.
 *   `-DGINKGO_HIP_AMDGPU="gpuarch1;gpuarch2"` the amdgpu_target(s) variable
     passed to hipcc for the `hcc` HIP backend. The default is none (auto).
+*   `-DGINKGO_BUILD_HWLOC={ON, OFF}` builds Ginkgo with HWLOC. If system HWLOC
```

Comment:
I have been thinking about this and now I think we should maybe set this to be `OFF` by default ? We might run into issues when inter-operating with libraries such as MFEM and deal.ii as they also have some hwloc support.

Reply:
Would we have issues? The big question is if they also download/install their own hwloc and cannot find ours. In another PR I fixed this (I think) as I found that we don't install the hwloc that we build whereas we should, considering that it's a public dependency.